docs/release-source/release/stubs.md

```diff
@@ -90,6 +90,8 @@ Note that it's usually better practice to stub individual methods, particularly
 
 Stubbing individual methods tests intent more precisely and is less susceptible to unexpected behavior as the object's code evolves.
 
+#### `var stubInstance = sinon.createStubInstance(MyConstructor, overrides);`
+
 If you want to create a stub object of `MyConstructor`, but don't want the constructor to be invoked, use this utility function.
```

Comment:
@fatso83 Yeah I wasn't even aware of that. Here's my knee-jerk thought: _on this line_, I think changing `sinon` to `sandbox` might confuse the reader more than it informs them: some readers may not know what `sandbox` is (yet) and assume they can't call this method on `sinon`. Maybe that's a good thing, but [the `sandbox` docs say](https://sinonjs.org/releases/v17/sandbox/):

> Unless you have a very advanced setup or need a special configuration, you probably only need to use that one in your tests for easy cleanup.

This could be far-fetched, but they might extrapolate that to mean `createStubInstance` is only for "very advanced setup or need a special configuration," too. To be honest, I'm not sure if that's a valid interpretation of `createStubInstance`, but I'm assuming it's not. I'm assuming it's used more often than non-`sinon` `sandbox`es. 

That said, I _do_ think this is useful information. Maybe it could be added as a detail below the header or in the link added by #2577 ?

I don't feel that strongly about anything I've said above: if you disagree with my take, I'm happy to make your suggested change.

Reply:
What about keeping the `sinon.create...` bits, but link to the `sandbox.md` docs for more elaborate examples?